subxt/src/config/mod.rs

```diff
@@ -44,14 +44,15 @@ pub trait Config: 'static {
         + Member
         + Default
         + Copy
-        + core::hash::Hash
-        + core::str::FromStr
+        + std::hash::Hash
+        + std::str::FromStr
```

Comment:
I know we are not targeting `no_std` yet but why not stick with `core`?

Reply:
Yeah I guess at the moment I wasn't a fan of the mnix of std and core imports and figured for now to just standardise to using std